### ISSUE TITLE
System timezone not being used.

### ISSUE THREAD

**Issue by justinjereza** [flathub/com.discordapp.Discord#80]
The app seems to be using PDT/PST instead of whatever is set for the system. This is not the case with the tarball downloaded directly from https://discordapp.com/.

**Comment by guihkx** [comment]
Yep, not working for me too. Here's a workaround for now:

```
flatpak override --user --env=TZ=America/Sao_Paulo com.discordapp.Discord
```

Replace `America/Sao_Paulo` with your timezone.

**Comment by lionirdeadman** [comment]
If the timezone workaround fixed it then the upstream bug (flatpak #3338) was fixed so it shouldn't happen again.
